src/app/constants.js

```diff
@@ -19,17 +19,64 @@ export const Component = {
 };
 
 export const DataType = {
+  OBS_INDEX: 'obsIndex',
+  OBS_EMBEDDING: 'obsEmbedding',
+  OBS_LOCATIONS: 'obsLocations',
+  OBS_FEATURE_MATRIX: 'obsFeatureMatrix',
+  FEATURE_INDEX: 'featureIndex',
+  OBS_SETS: 'obsSets',
+  IMAGE: 'image',
+  OBS_SEGMENTATIONS: 'obsSegmentations',
+  NEIGHBORHOODS: 'neighborhoods',
+  OBS_SET_GENOMIC_PROFILES: 'obsSetGenomicProfiles',
+  // Old data types
   CELLS: 'cells',
   CELL_SETS: 'cell-sets',
   EXPRESSION_MATRIX: 'expression-matrix',
   GENOMIC_PROFILES: 'genomic-profiles',
   MOLECULES: 'molecules',
-  NEIGHBORHOODS: 'neighborhoods',
   RASTER: 'raster',
 };
 
 
 export const FileType = {
+  // New file types
+  OBS_SETS_JSON: 'obsSets.json',
+  IMAGE_OME_ZARR: 'image.ome-zarr',
+  // Also for anndata-expression-matrix.zarr
+  OBS_INDEX_ANNDATA_ZARR: 'obsIndex.anndata.zarr',
```

Comment:
Commented on slack, but generalizing `xxxIndex` may be problemtaic in the case of missing indices....we may need a fallback or a different mechanism for specifying this.

Reply:
That is a good point, I just replied to you on slack and will update the PR(s)